crocket/src/main/java/com/crocket/CollisionHandler.java

```diff
@@ -27,15 +27,24 @@ public static boolean intersect(Hitbox hitbox1, Hitbox hitbox2) {
         int h2Width = hitbox2.getWidth();
         int h2Height = hitbox2.getHeight();
 
-        boolean isLeft = h1X < h2X + h2Width;
-        boolean isRight = h1X + h1Width > h2X;
-        boolean isAbove = h1Y < h2Y + h2Height;
-        boolean isBelow = h1Y + h1Height > h2Y;
-
-        return (isLeft &&
-                isRight &&
-                isAbove &&
-                isBelow);
+        //Checks collision from above
```

Comment:
I would suggest breaking these down to shorter if-statements and try to make it more readable